### PR TITLE
Update wording in DOCUMENTATION.org for readability and consistency.

### PR DESCRIPTION
**Pull request by danielhodson**
Minor updates to the spacemacs documentation to add consistency to "Notes" paragraphs and fix some grammar issues.

### CODE REVIEW

File: doc/DOCUMENTATION.org [hunk 1]
```diff
@@ -476,15 +476,15 @@ layers are not source controlled/.
...
+repository if you have one. Also, put your =~/.spacemacs= file in it.
 
-Then you are free to symlink your layers into =~/emacs.d/private= /or/ let them
-anywhere you want and reference the parent directory in the variable
+Then you are free to symlink your layers into =~/emacs.d/private= /or/ leave them
+anywhere you want, and reference the parent directory in the variable
 =dotspacemacs-configuration-layer-path= of your =~/.spacemacs=.
 
-Note that you could also have a dedicated repository for all your private layers
+*Note*: you could also have a dedicated repository for all your private layers
```
Comment (nixmaniack) [inline doc/DOCUMENTATION.org]: Capitalize `You`.

File: doc/DOCUMENTATION.org [hunk 2]
```diff
@@ -1280,12 +1280,12 @@ A color code is used for the battery status:
 | Discharging   | Orange |
 | Critical      | Red    |
 
-Note the these colors may vary depending on your theme.
+*Note*: these colors may vary depending on your theme.
```
Comment (nixmaniack) [inline doc/DOCUMENTATION.org]: Capitalize `These`.

File: doc/DOCUMENTATION.org [hunk 3]
```diff
@@ -476,15 +476,15 @@ layers are not source controlled/.
 *** Using an external Git repository
 This is the recommended way to manage your private layers.
 
-The best approach is to store all your private layers into an external Git
-repository. It is especially a good practice to store them in your =dotfiles=
-repository if you have one. Put also your =~/.spacemacs= file in it.
+The best approach is to store all your private layers in an external Git
+repository. It is good practice to store them in your =dotfiles=
```
Comment (nixmaniack) [inline doc/DOCUMENTATION.org]: Not a native english speaker but I find it odd to say `It is good practice` without an article. Should it be `It is a good practice`?

Comment (danielhodson) [reply doc/DOCUMENTATION.org]: I think you're right. I've updated the structure of the sentence to something which might read better.

**Comment by syl20bnr** [comment]
😍 I love those PRs, thank you for taking the time to improve the docs!
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**